Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "m_PathEaseTime", "type": "float32" },
+        { "name": "TargetDirection", "type": "vector3" },
+        { "name": "m_IsChangeCamera", "type": "bool" },
+        { "name": "m_IsPadCorrect", "type": "bool" },
+        { "name": "m_IsAutoChange2DPath", "type": "bool" },
+        { "name": "m_IsEnableFromFront", "type": "bool" },
+        { "name": "m_IsEnableFromBack", "type": "bool" }
+      ]
+    },
+    "ObjEggmanMedalSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjEggMaskBoxSpawner": {
+      "fields": [
+        { "name": "AttackInterval", "type": "float32" },
+        { "name": "SearchRadius", "type": "float32" },
+        { "name": "DeadTime", "type": "float32" },
+        { "name": "OffsetY", "type": "float32" },
+        { "name": "OffsetX", "type": "float32" }
+      ]
+    },
+    "ObjEggMaskCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" }
+      ]
+    },
+    "ObjMapPartsBaseSpawner": {
+      "fields": [
+        { "name": "IsPlayerTerrain", "type": "bool" },
+        { "name": "ForceGrid", "type": "bool" }
+      ]
+    },
+    "ObjMapPartsBoxSpawner": {
+      "parent": "ObjMapPartsBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "ObjMapPartsBoxSpawner::Type" },
+        { "name": "Width", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "Length", "type": "float32" }
+      ]
+    },
+    "ObjMapPartsPlaneSpawner": {
+      "parent": "ObjMapPartsBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "ObjMapPartsPlaneSpawner::Type" },
+        { "name": "Width", "type": "float32" },
+        { "name": "Height", "type": "float32" },
+        { "name": "Length", "type": "float32" }
+      ]
+    },
+    "ObjMapPartsMeshSpawner": {
+      "parent": "ObjMapPartsBaseSpawner",
+      "fields": [
+        { "name": "Type", "type": "ObjMapPartsMeshSpawner::Type" }
+      ]
+    },
+    "ObjMultiBoostBoxSpawner": {
+      "fields": [
+        { "name": "Revive", "type": "bool" }
+      ]
+    },
+    "ObjMultiPhantomBoxSpawner": {
+      "fields": [
+        { "name": "PixieType", "type": "ObjMultiPhantomBoxSpawner::PixieType" },
+        { "name": "PixieType2", "type": "ObjMultiPhantomBoxSpawner::PixieType2" },
+        { "name": "Revive", "type": "bool" }
+      ]
+    },
+    "ObjPunchingBallSpawner": {
+      "fields": [
+        { "name": "Length", "type": "float32" }
+      ]
+    },
+    "ObjSlotSpawner": {
+      "fields": [
+        { "name": "Probabilities[0]", "type": "float32" },
+        { "name": "Probabilities[1]", "type": "float32" },
+        { "name": "Probabilities[2]", "type": "float32" },
+        { "name": "Probabilities[3]", "type": "float32" },
+        { "name": "Probabilities[4]", "type": "float32" }
+      ]
+    },
```

Comment:
Parameter names based on in-game functionality and similar parameters specifically from `enmMotora`.